mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -788,20 +765,17 @@ ::mediapipe::Status GlAnimationOverlayCalculator::GlSetup() {
 
     void main() {
       // Sample the texture, retrieving an rgba pixel value
-      vec4 pixel = texture2D(texture, sampleCoordinate.xy);
+      vec4 pixel = texture2D(texture, sampleCoordinate);
       // If the alpha (background) value is near transparent, then discard the
       // pixel, this allows the rendering of transparent background GIFs
       if (pixel.a < 0.2) discard;
```

Comment:
This won't necessarily be at all related to world-space.  The sampleCoordinate only tells us where in our texture-map we need to look to color our pixel.  These textures usually are very compressed, and can be arbitrarily laid out in memory (like the head of the dino can be right next to its feet in the texture), so we have to use something else to get a pixel position.

We could derive something for this ourselves, but GLES gives us "gl_FragCoord", which I think should suffice.